R/transformCounts.R

```diff
@@ -564,12 +564,12 @@ setMethod("relAbundanceCounts", signature = c(x = "SummarizedExperiment"),
     }
     # Give warning if pseudocount should not be added
     # Case 1: only positive values
-    if( pseudocount != 0 && all(mat>0) ){
+    if( pseudocount != 0 && all(mat > 0) ){
```

Comment:
If missing values caused this, all() should also have na.rm?

Reply:
Just to make sure I added na.rm = TRUE, because:
```
> all(c(1, NA, 0) > 3)
[1] FALSE
> all(c(1, NA, 0) < 3)
[1] NA
> all(c(1, NA, 0) < 3, na.rm = TRUE)
[1] TRUE
```